### ISSUE TITLE
Add line to energy scatter plot

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/energy-skate-park#129]
Will make the plot and energies easier to read, while still giving a sense of speed to the skater.

**Comment by jessegreenberg** [comment]
The dots of the scatter plot fade away over time. Does the connecting line need to as well?

I am assuming the answer is yes, but this will be a little tricky to do in griddle so I want to make sure it is worth the time.

Alternatively, we could increase the frequency of adding points to the scatter plot so it is easier to see the path.

@arouinfar what do you think?

**Comment by jessegreenberg** [comment]
![image](https://user-images.githubusercontent.com/6396244/66328236-3f1f6180-e8fa-11e9-8c16-d424b8582b4e.png)

**Comment by arouinfar** [comment]
> The dots of the scatter plot fade away over time. Does the connecting line need to as well?
>
>I am assuming the answer is yes, but this will be a little tricky to do in griddle so I want to make sure it is worth the time.

That'd be the correct assumption @jessegreenberg, but let's hold off on that for now and try increasing the frequency of the dots. The screenshot in https://github.com/phetsims/energy-skate-park/issues/129#issuecomment-539084697 looks pretty good, but it might be nice be able to fine tune the frequency. Is this something you could add a query parameter for?

**Comment by jessegreenberg** [comment]
@arouinfar sounds good, the query parameter ?saveFrequency added for this ?saveFrequency=0.01 provides https://github.com/phetsims/energy-skate-park/issues/129#issuecomment-539084697

**Comment by arouinfar** [comment]
Awesome, thanks @jessegreenberg!

**Comment by jessegreenberg** [comment]
@arouinfar and I met for this:
- We are going to go with saveFrequency=0.01 now.
- We need to adjust the max limit for how many points are saved - this frequency increases the number of points saved so for low gravity and slow movement only a small portion of the skater path is plotted.
- There is an issue with how points fade away, they don't seem to all clear at the same time when skater direction changes and they should.

**Comment by jessegreenberg** [comment]
The second two points are actually pretty much the same issue - we need to change the way graphed skater sampelse are added and removed. At the moment skater samples are only added and removed based on amount of time. But they need to behave more like the skater samples in the measure screen. I am going to make a new issue for this because it isn't related to this one. Closing this because we decided on a solution for making the plot profile easier to understand.
